### DISCUSSION TITLE
Czy można ukryć zakończone oferty przez REST API?

### DISCUSSION THREAD

**Discussion by net-shops** [allegro/allegro-api#4330] [Suggestion - verification]
Hej,

Czy jest opcja ukrycia ofert przez REST API tak samo jak przez zaznaczenie oferty i kliknięcie "UKRYJ" na menu na dole przez UI?

<img width="939" alt="Screenshot 2021-02-04 at 00 02 21" src="https://user-images.githubusercontent.com/73105269/106820878-6d42bc00-667c-11eb-910d-f00bca407357.png">

**Comment by MaciejFrackowiak** [comment]
Nie, w API nie ma takiej funkcjonalności - dostępna jest wyłącznie na stronie WWW.

**Comment by net-shops** [comment]
A planujecie to wdrożyć? Jest to, nie ukrywam, spory problem - zaśmiecanie listy ofert elementami, które już nigdy nie będą wznawiane.

**Comment by MaciejFrackowiak** [comment]
Mamy to na liście sugestii dot. rozwoju zasobów REST API - dodaję Twój głos.

**Comment by Aryak2099** [comment]
Przecież to możesz zrobić sobie programowo bez problemu w swojej aplikacji, więc jaki jest sens robienia tego tutuaj, my robimy tak że zmieniamy nazwę sygnatury na "EMPTYOFFER" i nasz program nie pokazuje wtedy ofert z taką sygnaturą , lub mozna stowrzyć biblioteke z numerami ofert i wtedy też je wykluczać.

**Reply by hahyes** [reply]
> więc jaki jest sens robienia tego tutuaj

Po co pobierać w ogóle oferty, których się nie chce?
